pkg/webservice/handlers.go

```diff
@@ -277,6 +277,9 @@ func getPartitionJSON(partition *scheduler.PartitionContext) *dao.PartitionDAOIn
 func getApplicationJSON(app *objects.Application) *dao.ApplicationDAOInfo {
 	allocations := app.GetAllAllocations()
 	allocationInfos := make([]dao.AllocationDAOInfo, 0, len(allocations))
+	placeholderInfos := make([]dao.PlaceholderDAOInfo, 0, len(app.PlaceholderDatas))
```

Comment:
This needs to directly come from the application similar to the way we process allocations: `app.GetPlaceholderDatas()`